### DISCUSSION TITLE
Does it work for js dos files

### DISCUSSION THREAD

**Discussion by therandomspoon** [git-lfs/git-lfs#5264] [Q&A] [answered]
It’s for my DOS box project

**Comment by bk2204** [comment, marked as answer]
Hey,

Git LFS can store any type of file within it, up to pretty unreasonable sizes (2^63-1 bytes).  Very large files (many GB) can perform poorly, though, since it can take some time to hash data.  So it will work just fine to store any type of file whatever.

However, Git LFS does not run on DOS, since Go, the language Git LFS is implemented in, does not run on DOS, so you may want to do development on another operating system and copy your files over.  Note that Git doesn't work on DOS, either, as far as I know.

If your JS files are small, then you may not even need Git LFS, and storing those files as regular Git files may be easier to work with.
